src/app/frontend/deploy/deploy.html

```diff
@@ -45,7 +45,8 @@ <h3 class="md-headline kd-deploy-form-title">Deploy a Containerized App</h3>
         </deploy-from-file>
       </div>
 
-      <md-button class="md-raised md-primary" type="submit" ng-disabled="ctrl.isDeployDisabled()">
+      <md-button id="kd-deploy-btn" class="md-raised md-primary" type="submit"
```

Comment:
Can you convert the ID to a class? It is a good practise to never use IDs, unless you really really need them :) With that practise in mind you'll avoid problems when someone adds second deploy button here (which has pretty generic meaning/name, yea?)

Sorry for nagging.